docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,29 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it makes requests to GitHub API to check for new version and potentially download the latest tarball of the Minishift or OpenShift client tool `oc`.
```

Comment:
hmm, as said, we are not downloading a tarball all the time. On Windows for example it is a zip. Hence I tried to avoid the use of _tarball_.